Specs/Scene/ConditionalExpressionSpec.js

```diff
@@ -24,13 +24,22 @@ defineSuite([
     };
 
     var jsonExp = {
-        conditional : {
+        conditions : {
             '${Height} > 100' : 'color("blue")',
             '${Height} > 50' : 'color("red")',
             'true' : 'color("green")'
         }
     };
 
+    var jsonExpWithExpression = {
```

Comment:
Looks good.  There should also be a test to verify that `${expression} === undefined` when `expression` is not there, and a check to verify that `evaluate` returns undefined when `conditions` is not there.  These are some of the corners in the schema that I alluded to in https://github.com/AnalyticalGraphicsInc/3d-tiles/pull/80